airbyte-webapp/src/views/Connection/CatalogTree/components/PathPopout.tsx

```diff
@@ -81,11 +72,9 @@ export const PathPopout: React.FC<PathPopoutProps> = (props) => {
       placeholder={props.placeholder}
       components={props.isMulti ? { MultiValue: () => null } : undefined}
       targetComponent={({ onOpen }) => (
-        <div onClick={onOpen}>
+        <PathPopoutButton items={props.isMulti ? props.path?.map(pathDisplayName) : props.path} onClick={onOpen}>
```

Comment:
Nice refactor, this looks so much better in the UI now! I had thought those dropdowns looked bad for a long while now :)

Reply:
There's a new version of this table in the works, but I figured I give it a little ❤️ until we get there.